### DISCUSSION TITLE
Adding selection

### DISCUSSION THREAD

**Discussion by Randalix** [helix-editor/helix#4369] [Q&A] [answered]
I did the tutor, but I'm missing / not getting how to select two different words.

 --> Everybody wants to be a bat,

How can I select Everybody and bat? My goal is to switch 2 words wich is done by Alt-( I think. But I was not able to try that yet, because I can't select both at the same time.

Thanks in advance for your help .

**Comment by the-mikedavis** [comment, marked as answer]
For this case you could select the line and subselect (`s`) on the regex `(Everybody|bat)`.

Marks (#3720) would be another way to approach this. On that branch you could select "Everybody" with any sort of navigation, use `^s` to save that selection to the marker register, select "bat" with any sort of navigation, then use `^ca` to append the saved selection.

**Reply by dbarnett** [reply]
I've used some multi-cursor editors where you could <kbd>ctrl</kbd>+click (with the mouse) to add an additional cursor. Would a paradigm like that ever be an option, where you could place the cursor on "Everybody", <kbd>ctrl</kbd>+click on "bat", and then type <kbd>e</kbd> to select both words to end-of-word?

**Reply by the-mikedavis** [reply]
That is already implemented: you can use <kbd>alt</kbd> and click and drag to create new selections

**Reply by dbarnett** [reply]
Hmm, is there any setting that could be related, or some terminals that work better than others for <kbd>alt</kbd> modifier? I can't seem to get that to work, <kbd>alt</kbd>+click just moves the cursor to the beginning of whatever line I click on instead of adding a selection.

**Reply by the-mikedavis** [reply]
Alt typically needs to be enabled when working on macOS https://github.com/helix-editor/helix/issues/2469

**Reply by dbarnett** [reply]
Oh thanks, I was trying to use it on a Chromebook actually, but that apparently has a similar issue swallowing alt-click: https://support.google.com/chromebook/thread/14888778/how-do-i-disable-the-alt-click-from-opening-the-right-click-context-menu-on-the-os-and-websites. Is that shortcut configurable by any chance, or would it be possible to provide an alternate key through e.g. the space menu for that?

**Comment by Randalix** [comment]
Thanks for the quick answer and the great work. Really like the project <3
